lib/CMakeLists.txt

```diff
@@ -70,4 +70,5 @@ if(CONFIG_HAS_STM32LIB)
     target_link_libraries(app PUBLIC stm32wba_ll_lib)
 
   endif()
+  add_subdirectory_ifdef(CONFIG_BT_STM32WB0 stm32wb0)
```

Comment:
This is fine but we need consistency across vraious libs.
Hence I'd expect the same to be done fro WBA and WB

Reply:
I didn't get this comment well. Should I do something?

Reply:
You're adding a sub section to handle WB0 case, which is indeed a good idea.
But in that case, for consistency, it would be nice to do the same on other series (in a dedicated commit).